samila/functions.py

```diff
@@ -1,7 +1,11 @@
 # -*- coding: utf-8 -*-
 """Samila functions."""
-
+from typing import Callable, Tuple, Dict, Generator, List
+from typing import Any, Union, Iterable
 import sys
+import matplotlib.axes
```

Comment:
Why do you import `matplotlib.axes`, `matplotlib.colors`, and `matplotlib.figure` again? We have already imported `matplotlib`.

Reply:
Ah, that's auto import probably. will fix it now